src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const shortPeriod = 3;
+  const longPeriod = 7;
+  const discountForShortPeriod = 20;
+  const baseCost = 40;
+  const discountForLongPeriod = 50;
+  const totalAmountOne = days * baseCost;
+  const totalAmountTwo = days * baseCost - discountForShortPeriod;
+  const totalAmountThree = days * baseCost - discountForLongPeriod;
```

Comment:
ти тут робиш наперед обчислення, хоча в кращому випадку 2 з них вийдуть непотрібні (якйщо спрацює перший if)
```suggestion
  const baseAmount = days * baseCost;
```